packages/webapp/pages/onboarding.tsx

```diff
@@ -92,6 +100,7 @@ export function OnboardPage(): ReactElement {
   };
 
   const formRef = useRef<HTMLFormElement>();
+  const [onSelectTopic, setOnSelectTopic] = useState(false);
```

Comment:
Let's reserve the `on` prefix on names for functions/handlers. I think we can swap the `hasSelectedTopics` to this one.